docs/usage/cmd_show.rst

```diff
@@ -2,16 +2,19 @@
 Showing Application Details
 ===========================
 
-``shpkpr show`` allows you to view details of a single application that is currently deployed to Marathon::
+``shpkpr show`` allows you to view details of one or more applications that are currently deployed to Marathon::
 
     $ shpkpr show --help
     Usage: shpkpr show [OPTIONS]
 
-      Shows detailed information for a single application.
+      Shows detailed information for one or more applications.
```

Comment:
Minor: Might be more helpful if this read as `... one or _all_ applications`. I didn't realise you couldn't request info for multiple specific applications until I read down.